docs/atoz.md

```diff
@@ -0,0 +1,487 @@
+~~~
+
+Once successfully installed the application should be available in your local user's `PATH`. To test the installation was successful please use this command:
+
+~~~bash
+wetty -h
+~~~
+
+### Accessing the web interface.
+
+This needs to be done here because it is not easy to do in the next steps if `wetty` is running in the terminal.
+
+This command will generate the correct URL you need to visit after using the start up commands in the following section.
+
+~~~bash
+echo https://$(curl -s4 icanhazip.com):3000
+~~~
+
+*Please make make a note of this URL now.*
+
+### Running wetty
+
+Now we have all the ground work done we can focus on our `wetty` server configuration settings.
+
+For example, the below command would provide a `https` instance with automatic `ssh` authorisation using our `wetty` private key on port `3000` accessible at `https://IP:3000` . Refer to the previous URL generation command.
+
+**Important note:** This command will run in your current terminal session and not in the background.
+
+~~~bash
+wetty --host 0.0.0.0 -p 3000 --title wetty --base / --sshkey ~/.ssh/wetty --sshhost localhost --sshuser $(whoami) --sshport 22 --sshauth publickey --sslkey ~/.ssl/wetty.key --sslcert ~/.ssl/wetty.crt
+~~~
+
+#### forever to manage wetty
+
+Now you can use `forever` we installed to run `wetty` in the background instead of directly in your terminal
+
+~~~bash
+forever start ~/bin/wetty --host 0.0.0.0 -p 3000 --title wetty --base / --sshkey ~/.ssh/wetty --sshhost localhost --sshuser $(whoami) --sshport 22 --sshauth publickey --sslkey ~/.ssl/wetty.key --sslcert ~/.ssl/wetty.crt
+~~~
+
+To stop `wetty` from running you can use this command
+
+~~~bash
+forever stop ~/bin/wetty
+~~~
+
+#### Optional - config file.
+
+**Important Note:** This is an optional and unsupported method to load settings from a physical configuration file. It's a hack and you need to accept it has some limitations but I've made it as easy as I can to use.
+
+Since `wetty` does not have configurations files and all commands are passed as command line arguments we can fake this behaviour by using a method to read from a text file and expand variables then pass this to `wetty`. All arguments must be on a single line
+
+Create a directory to store our configuration data using this command:
+
+~~~bash
+mkdir -p ~/.config/wetty
+~~~
+
+Now populate our `config` file with some settings. This examples is the same command as above.
+
+~~~bash
+echo -n '--host 0.0.0.0 -p 3000 --title wetty --base / --sshkey ~/.ssh/wetty --sshhost localhost --sshuser $(whoami) --sshport 22 --sshauth publickey --sslkey ~/.ssl/wetty.key --sslcert ~/.ssl/wetty.crt' > ~/.config/wetty/config
+~~~
+
+This configuration file is now available here for you to manage your settings.
+
+~~~bash
+~/.config/wetty/config
+~~~
+
+Now we can load this file as part of the command we pass to `wetty` with shell expansion and command substitution.
+
+~~~bash
+wetty $(eval echo $(cat ~/.config/wetty/config))
```

Comment:
Wetty supports using ENV variables. 
Why not load an env file?

Reply:
I was not aware of this. I missed it somehow but it makes sense to document this.

I will test it and modify accordingly.